packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -502,6 +542,13 @@ class InspectorRowContent extends StatelessWidget {
                   ),
                 ),
               ),
+              if (InspectorTreeControllerFlutter
+                      .enableExperimentalStoryOfLayout &&
+                  controller.animationController != null)
+                ConstraintsDescription(
+                  animationController: controller.animationController,
```

Comment:
I would flow in the animationController as a named parameter to the InspectorRowContent from its parent rather than grabbing it off the controller.